src/app/guards/oauth.guard.ts

```diff
@@ -35,23 +35,32 @@ export class OauthGuard implements CanActivateChild {
         .declareOauthSession(<DeclareOauthSession>next.queryParams)
         .pipe(
           map((value) => {
-            if (value.errors.length) {
-              console.log('The backend cant declare the oauth section ')
+            if (value.errors.length || value.error) {
               // redirect the user to the login if something goes wrong on the backend declaration
-              // TODO @leomendoza123 Throw toaster error
+              // TODO @leomendoza123 Throw toaster error and display error description
               return this._router.createUrlTree(['/signin'])
             } else if (!value.userOrcid || !value.userName) {
               // The users is not logged in
               if (state.url.startsWith('/oauth/authorize')) {
+                // todo validate if it works on ie11
```

Comment:
Please assign this todo to someone or resolve if possible

Reply:
Fixed!